.github/workflows/docker_publish.yml

```
@@ -194,7 +191,7 @@ jobs:
 
 
     container:
-      image: ghcr.io/${{ github.repository_owner }}/ubuntu_20.04_py3${{ matrix.hdf5 }}${{ matrix.dagmc }}${{ matrix.pymoab }}${{ matrix.openmc }}_pyne-deps:ci_testing
+      image: ${{ matrix.env.DOCKER_IMAGE_BASENAME }}${{ matrix.hdf5 }}${{ matrix.dagmc }}${{ matrix.pymoab }}${{ matrix.openmc }}_pyne-deps:ci_testing
```

```suggestion
      image: ${{ env.DOCKER_IMAGE_BASENAME }}${{ matrix.hdf5 }}${{ matrix.dagmc }}${{ matrix.pymoab }}${{ matrix.openmc }}_pyne-deps:ci_testing
```

I think this is causing the build test to fail

---

That's strange... I think I tried it without the `matrix` before and the workflow wouldn't run at all, saying that "env is unrecognized". This was in the workflow run that happened with the `fix basename variable references` commit.

---

I'd be curious to see that error

---

Here's the link to the workflow with the error:
https://github.com/bquan0/pyne/actions/runs/3251869373/workflow

---

That is definitely different from what I expect.  I guess it might make sense since the environment hasn't been established yet - we are defining the container in which we will creating the environment.

However, that variable is also not defined in the matrix.  Maybe we need to also define a variable that is in scope within the Github action?

---

note: it should NOT be added to the matrix, I don't think

---

In fact, we might just want to reuse the original variables in place rather than defining a new variable to be used in a single place.

---

Looks like reusing the original variable solved the problem.